flipperdevices/flipperzero-firmware
# Picopass: dump full card, extract some details

# What's new

- Include a lot more info in picopass
- parse more parts of the picopass (ideally they'll show up in the UI when I figure how how best to make them look)

# Verification 

- scan iclass
- save
- view saved file
- see more info than before

# Checklist (For Reviewer)
